src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -41,10 +41,10 @@
     <PackageReference Include="Microsoft.Azure.Functions.JavaWorker" Version="1.5.2-SNAPSHOT" />
     <PackageReference Include="Microsoft.Azure.Functions.NodeJsWorker" Version="1.1.1" />
     <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker" Version="1.0.194" />
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.14" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions" Version="3.0.5" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.14-11665" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions" Version="3.0.4" />
```

Comment:
Make sure Extensions is back to 3.0.5 (you probably didn't have the latest locally).

Reply:
I'd merged dev into my topic branch recently; set it back to 3.0.5 and my build failed locally because of the reference to the updated SDK version I was importing (since preview is regarded as a downgrade from 3.0.14). Would the solution to this issue be to deploy a new major version of the SDK (3.0.15)?

Reply:
This still needs to go back to 3.0.5, right?